eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/MessageQueue.java

```diff
@@ -54,22 +57,23 @@ public MessageQueue(int capacity) {
         }
         this.items = new MessageEntity[capacity];
         this.lock = new ReentrantLock();
-        this.notEmpty = lock.newCondition();
-        this.notFull = lock.newCondition();
+        this.isEmpty = lock.newCondition();
+        this.isFull = lock.newCondition();
     }
```

Comment:
These modification will confuse the developer or user. We name it `notFull` in situation of waiting on a full queue until not full, `notEmpty`  in situation of waiting on a empty queue util not empty. This naming method comes from jdk. https://github.com/openjdk/jdk/blob/a0f4a948bb76ce5ea11d8fe08e788ed43516ca4e/src/java.base/share/classes/java/util/concurrent/ArrayBlockingQueue.java#L370